lib/Doctrine/ODM/PHPCR/DocumentManagerInterface.php

```diff
@@ -525,4 +525,15 @@ public function close();
      * @throws PHPCRException                if $document is not managed
      */
     public function getNodeForDocument($document);
+
+    /**
+     * Evaluates the documents identifier. Will return null when document is not managed.
```

Comment:
the "will return null when document is not managed" is no longer correct. i would remove it and rely on the `@throws` annotation.

maybe say: "Get the document identifier phpcr-odm is using."